packages/almin/src/UILayer/StoreGroupTypes.ts

```diff
@@ -1,17 +1,14 @@
 // MIT © 2017 azu
 import { Store } from "../Store";
+// Apply: StoreMap -> StateMap
 // { stateName: Store }
-export type StoreMap<T> = {
-    // T[P] as Store<State>
-    // Now, State is T[P}
-    [P in keyof T]: Store<T[P]>
-};
```

Comment:
@azz Ok. I've filed it as issue 
https://github.com/prettier/prettier/issues/2462